com.unity.render-pipelines.universal/ShaderLibrary/Deprecated.hlsl

```diff
@@ -6,4 +6,8 @@
 #define SCREENSPACE_TEXTURE_FLOAT   TEXTURE2D_X_FLOAT
 #define SCREENSPACE_TEXTURE_HALF    TEXTURE2D_X_HALF
 
+// Typo-fixes, re-route to new name for backwards compatiblity (if there are external dependencies).
+#define kDieletricSpec kDielectricSpec
```

Comment:
For next PR: we have an shader upgrader tool, it just hit me we could ugprade users shaders.